views/logIn.hbs

```diff
@@ -0,0 +1,18 @@
+<form action="/login" method="POST" id="form">
```

Comment:
El problema por el que no puedes acceder a las sesiones de **main** y **private** es que el **_action_** del formulario no está apuntando bien a la ruta LOGIN. Es _**"/log-in"**_ y por tanto no estás creando la **_session_**.

`<form action="/log-in" method="POST" id="form">`